### ISSUE TITLE
Before #5: Delete numgroup and make functions self contained

### ISSUE THREAD

**Issue by abstractmachines** [devinenoise/hello-world#11]
This was a part of #8 that didn't get completed. @devinenoise 

NumGroup is "used to loop through a function" or something like that - let's delete that code because anything that a function is supposed to do/operations should stay solely inside of that function. 

Basically if we need to loop inside of a function, all of that should happen inside the scope (curly braces) of a function. 

I would address this really soon before #5, because the numGroup thing will probably get in the way of us writing clean, easy to understand code!

**Comment by abstractmachines** [comment]
What this should look like is this:

```
void numberArray() {

   /* anything that "numgroup did before" should all be in here. 
      numGroup should not be used in this function because it's not inside the "scope" (curly braces) of the function.
  */
    for (int i = 0; i < 5; i++) {

      
        cout << numGroup[i] << endl; // probably want to define numGroup here first.


    }
}
```

What I suggest is:

```
void numberArray() {
    int numGroup[5] = { 1, 3, 5, 7, 9 };
    for (int i = 0; i < 5; i++) {
        cout << numGroup[i] << endl;
    }
}
```

Now, what happens if numGroup happens to be more than 5? Won't that make our code break?

So how about:

```
void numberArray() {
    int numGroup[5] = { 1, 3, 5, 7, 9 };
    for (int i = 0; i < LENGTH OF OUR ARRAY; i++) {
        cout << numGroup[i] << endl;
    }
}
```

Length of array in C++: https://www.tutorialspoint.com/how-do-i-find-the-length-of-an-array-in-c-cplusplus

**Comment by devinenoise** [comment]
I've updated the code using sizeof() inside of the numberArray function.

**Comment by abstractmachines** [comment]
Resolved mostly, by #10
